frontend/src/components/profile/profileForm/index.jsx

```diff
@@ -16,11 +14,23 @@ const ProfileForm = ({ profile, onSave, onCancel }) => {
             last_name: lastName,
             email,
             phone,
-            isDonor,
-            isStudent,
+            // Remove isDonor and isStudent from the save payload
         });
     };
 
+    // Determine user status based on isDonor and isStudent
+    const getStatus = () => {
+        if (profile.isDonor && profile.isStudent) {
+            return 'Donor & Student';
+        } else if (profile.isDonor) {
+            return 'Donor';
+        } else if (profile.isStudent) {
+            return 'Student';
+        } else {
+            return 'None';
+        }
+    };
+
```

Comment:
Since "Student" is the special group because we have restrictions, you can be a `student`, a `donor`, or `both`, but `never None`. Every email we cannot verify as a `student `will automatically be assigned the Donor status